website/content/en/preview/provisioner.md

```diff
@@ -144,6 +147,18 @@ Karpenter supports `amd64` nodes, and `arm64` nodes.
 
 Karpenter supports specifying capacity type, which is analogous to [EC2 purchase options](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-purchasing-options.html).
 
+### Allocation Strategy
+
+- key: `karpenter.sh/allocation-strategy`
+
+☁️ **AWS**
+
+- values
+  - `capacity-optimized` (default for spot)
```

Comment:
Actually, the default for Spot right now is capacity-optimized-prioritized right now, which is prioritized by instance size (lowest size gets highest priority)

Reply:
Thank you for the review about the default of spot! The issue I created ( #1240 ) reflects the fact.